niap-cc/NIAPSecExample/app/src/main/java/com/android/certifications/niap/CustomSocketFactory.java

```diff
@@ -0,0 +1,562 @@
+package com.android.certifications.niap;
```

Comment:
Nit: new files should have the following copyright statement:

```
/*
 * Copyright 2022 The Android Open Source Project
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *      http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */
```

Reply:
added

Reply:
Thanks for addressing this! However, when making changes to existing code with a copyright statement, the original copyright year must be retained.

Reply:
Thanks, will fix it.

Reply:
I noticed several files still show the change from 2020 to 2022; did the commit that reverted those back to 2020 make it into this pull request?

Reply:
revert these as much as possible